source/_addonStore/dataManager.py

```diff
@@ -142,10 +157,10 @@ def _getCachedAddonData(self, cacheFilePath: str) -> Optional[CachedAddonsModel]
 			cacheData = json.load(cacheFile)

```

Comment:
Could you also look at fixing #15077,  at the same time as https://github.com/nvaccess/nvda/pull/15071/files#r1249915944

Reply:
@nvdaes, you have asked me to test this PR in https://github.com/nvaccess/nvda/issues/15077#issuecomment-1617464139.

Testing from source on latest commit (dfe1b229fb9564c79d36769d5775f9ca514c901e) with the [file](https://github.com/nvaccess/nvda/files/11907168/_cachedCompatibleAddons.json.txt) from #15077, the issue is still present.

You should add a try/except around the two following lines in `_getCachedAddonData:`
```
 		with open(cacheFilePath, 'r') as cacheFile:
			cacheData = json.load(cacheFile)
```